third_party/move/move-compiler-v2/tests/checking/dotdot/assign.exp

```diff
@@ -0,0 +1,10 @@
+
+Diagnostics:
+error: unexpected token
```

Comment:
All test cases in this file are parser errors, which are meant to showcase that assignment is not supported yet. I split it into two, one for positional fields and one for named fields.

Reply:
@fEst1ck If all test cases are parse errors, then why is only one error reported in the diagnostics? Should there not be multiple errors reported?

Reply:
@fEst1ck So all the test code after the first parse error is completely useless - sure, the cause of the errors maybe the same conceptually, but the purpose of the test case is to check it and demonstrate it in exp files. Otherwise, we have the illusion that the other cases are being tested, but we see no evidence of it - a code change that affects the rest will not be caught by the test case as the first parse error hides them.